grouper/models/user.py

```diff
@@ -0,0 +1,82 @@
+import logging
+
+from sqlalchemy import Boolean, Column, Integer, String
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.orm import relationship
+
+from grouper.constants import GROUP_ADMIN, MAX_NAME_LENGTH, PERMISSION_ADMIN, USER_ADMIN
+from grouper.models.audit_log import AuditLog
+from grouper.models.base.model_base import Model
+from grouper.models.comment import CommentObjectMixin
+from grouper.models.counter import Counter
+from grouper.models.user_token import UserToken
+from grouper.plugin import get_plugins
```

Comment:
it'd be nice to avoid these cross model dependencies if possible. mostly that looks like pull the function into a grouper level module.

In the case of `UserToken` that might just be changing the reference to a string.
```
tokens = relationship('user_token', back_populates="user")
```